backend/src/config/env.validation.ts

```diff
@@ -3,49 +3,72 @@ import {
   IsEnum,
   IsNotEmpty,
   IsNumber,
+  IsOptional,
   IsString,
   validateSync,
 } from 'class-validator';
 
-enum Environment {
+export enum EnvironmentEnum {
   Development = 'development',
   Production = 'production',
   Test = 'test',
-  Provision = 'provision',
-  Local = 'local',
+  CI = 'ci',
 }
 
 class EnvironmentVariables {
-  @IsEnum(Environment)
-  NODE_ENV: Environment;
+  @IsNotEmpty()
+  @IsEnum(EnvironmentEnum)
+  NODE_ENV: EnvironmentEnum;
 
+  @IsOptional()
   @IsString()
   DATABASE_HOST: string;
+  @IsOptional()
   @IsNumber()
   DATABASE_PORT: number;
+  @IsNotEmpty()
   @IsString()
   DATABASE_NAME: string;
+  @IsNotEmpty()
   @IsString()
   DATABASE_USER: string;
+  @IsNotEmpty()
   @IsString()
   DATABASE_PASSWORD: string;
 
+  @IsOptional()
   @IsString()
   INTRA_AUTH_URL: string;
+  @IsOptional()
   @IsString()
   INTRA_TOKEN_URL: string;
+  @IsOptional()
   @IsString()
   INTRA_FETCH_URL: string;
+  @IsOptional()
   @IsString()
   INTRA_REDIRECT_URL: string;
+
   @IsNotEmpty()
   @IsString()
   INTRA_CLIENT_ID: string;
   @IsNotEmpty()
   @IsString()
   INTRA_CLIENT_SECRET: string;
+  @IsNotEmpty()
   @IsString()
   SESSION_SECRET: string;
+
+  @IsNotEmpty()
+  @IsString()
+  TFA_SECRET_KEY: string;
+
+  @IsOptional()
+  @IsString()
+  AWS_ACCESS_KEY_ID: string;
+  @IsOptional()
+  @IsString()
+  AWS_SECRET_ACCESS_KEY: string;
```

Comment:
```suggestion
  @IsNotEmpty()
  @IsEnum(EnvironmentEnum)
  NODE_ENV: EnvironmentEnum;

  @IsOptional()
  @IsString()
  DATABASE_HOST: string;

  @IsOptional()
  @IsNumber()
  DATABASE_PORT: number;

  @IsNotEmpty()
  @IsString()
  DATABASE_NAME: string;

  @IsNotEmpty()
  @IsString()
  DATABASE_USER: string;

  @IsNotEmpty()
  @IsString()
  DATABASE_PASSWORD: string;

  @IsOptional()
  @IsString()
  INTRA_AUTH_URL: string;

  @IsOptional()
  @IsString()
  INTRA_TOKEN_URL: string;

  @IsOptional()
  @IsString()
  INTRA_FETCH_URL: string;

  @IsOptional()
  @IsString()
  INTRA_REDIRECT_URL: string;

  @IsNotEmpty()
  @IsString()
  INTRA_CLIENT_ID: string;

  @IsNotEmpty()
  @IsString()
  INTRA_CLIENT_SECRET: string;

  @IsNotEmpty()
  @IsString()
  SESSION_SECRET: string;

  @IsNotEmpty()
  @IsString()
  TFA_SECRET_KEY: string;

  @IsOptional()
  @IsString()
  AWS_ACCESS_KEY_ID: string;

  @IsOptional()
  @IsString()
  AWS_SECRET_ACCESS_KEY: string;
```
espacinho, felas? :pinching_hand: